include/exadg/utilities/print_general_infos.h

```diff
@@ -142,6 +116,7 @@ print_general_info(dealii::ConditionalOStream const & pcout,
   if(not(is_test))
   {
     print_dealii_info(pcout);
+    print_exadg_info(pcout);
```

Comment:
I would not print ExaDG info between deal.II and matrix-free info. Maybe we print ExaDG first, given that its the ExaDG software here and given that deal.II/matrix-free is additional third-party information.

Reply:
Done :)